### ISSUE TITLE
Improve README

### ISSUE THREAD

**Issue by mayberryrfd** [mayberryrfd/rs-pugbot#5]

**Comment by onelson** [comment]
The biggest need I can see immediately is some info about setup for developers. I'm assuming you need to hook this up to discord _somehow_ in order to get working.

**Comment by mayberryrfd** [comment]
Yes, great place to improve that. But so far it is not really necessary to connect to Discord at all in dev. 

The serenity crate guarantees an interface to the Discord API so i just have to code to serenity and make sure the biz logic I write works via tests.
